src/gas/GasFeeController.test.ts

```diff
@@ -74,6 +76,9 @@ describe('GasFeeController', () => {
         estimatedBaseFee: '28',
       })
       .persist();
+    mockGasFeeRequest.on('request', () => {
```

Comment:
This was the best I could come up with for tracking number of requests. I look through nock documentation fairly extensively, tried a few other things, but this meets my exact need. Open to suggested alternative if there is a simpler way based on the nock api.

Reply:
I looked this up and it seems event is the way to go, however (nit) I'd change the event handler to a jest mock and use `expect(handler).toHaveBeenCalledTimes` instead of a counter

Reply:
Thanks for the suggestion @wachunei, I made this change in the latest commit 4f8ae4b2